### PR TITLE
fix: Props

### PR DESCRIPTION
**Pull request by AnnDv**

### CODE REVIEW

File: reactMovieApp/src/App.js [hunk 1]
```diff
@@ -1,29 +1,12 @@
...
-import { CONFIG } from './config';
-import MovieCard from './components/MovieCard';
 import React from 'react';
-import { useState, useEffect } from 'react';
-import "./App.css"
-
-const API_URL = CONFIG.BASE_URL + CONFIG.POPULAR_MOVIE_URL + CONFIG.API_KEY;
+import FetchMovies from './api/FetchMovies';
 
 function App() {
```
Comment (nkv843) [inline reactMovieApp/src/App.js:5]: maybe arrow func?

File: reactMovieApp/src/App.js [hunk 2]
```diff
@@ -1,29 +1,12 @@
...
-  }, [])
-
   return (
-    <div className='container'>
-      <MovieCard movies={movies}/>    
-    </div>
-    
+    <>
+      <FetchMovies />
+    </>
```
Comment (nkv843) [inline reactMovieApp/src/App.js:9]: why not just `<FetchMovies/> ` ?

File: reactMovieApp/src/api/FetchMovies.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { CONFIG } from "../config";
+import React, { useEffect, useState } from "react";
+import MovieCard from "../components/MovieCard";
+
+const API_URL = CONFIG.BASE_URL + CONFIG.POPULAR_MOVIE_URL + CONFIG.API_KEY;
+
+export const FetchMovies = () => {
```
Comment (nkv843) [inline reactMovieApp/src/api/FetchMovies.js:7]: first export

File: reactMovieApp/src/api/FetchMovies.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  
+  return (
+    <div className='container'>
+      <MovieCard movies={movies}/>    
+    </div>
+      
+  );
+}
+
+export default FetchMovies;
```
Comment (nkv843) [inline reactMovieApp/src/api/FetchMovies.js:27]: second export

File: reactMovieApp/src/components/MovieList.js [hunk 5]
```diff
@@ -1,9 +1,9 @@
...
-import MovieTitle from './MovieTitle';
-import "./MovieCard.css";
-import ShowInfoButton from './ShowInfoButton';
+import MovieImage from './movieComponents/MovieImage';
+import MovieTitle from './movieComponents/MovieTitle';
+import "./MovieList.css";
+import ShowInfoButton from './movieComponents/ShowInfoButton';
 import PropTypes from 'prop-types';
 
 function MovieList({movies}){
```
Comment (nkv843) [inline reactMovieApp/src/components/MovieList.js:9]: i think you can extract 

> `<div key={movie.id} className="card_description">`
   `  <MovieImage item={movie}/>`
   `  <MovieTitle item={movie}/>`
   `  <ShowInfoButton item={movie}/>`
`</div>`

as component movieItem with { movie } props

File: reactMovieApp/src/components/MovieCard.js [hunk 6]
```diff
@@ -1,25 +1,14 @@
...
-			))}
-			
-		</div>
+function MovieCard({movies}){
+  return (
+	<div>
+		<MovieList movies={movies}/>
+	</div>
 	);
 }
```
Comment (dorin-js) [inline reactMovieApp/src/components/MovieCard.js:12]: should be same 2-4 space indentation

File: reactMovieApp/src/components/movieComponents/MovieDescription.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+const MovieDescription = (movie) => {
+    const info = movie.item;
+    
+    return (
+    <div className='text'>
+        <p>{info.overview}</p>
+    </div>
+    )
+}
```
Comment (dorin-js) [inline reactMovieApp/src/components/movieComponents/MovieDescription.js:13]: you can destructure movie.item and return inline
const MovieDescription = ({ item: info }) => (
    <div className='text'>
        <p>{info.overview}</p>
    </div>
)

Comment (dorin-js) [inline reactMovieApp/src/components/movieComponents/MovieDescription.js:13]: you can destructure movie.item and return inline
`
const MovieDescription = ({ item: info }) => (
    <div className='text'>
        <p>{info.overview}</p>
    </div>
)
`

**Pull request merged**